draft-wussler-openpgp-pqc.md

```diff
@@ -1205,8 +1205,8 @@ consists of:
 
 ### Key Material Packets
 
-The composite Dilithium + ECC schemes MUST be used only with v5 keys, as
-defined in [I-D.ietf-openpgp-crypto-refresh] Section 5.2.2.
+The composite Dilithium + ECC schemes MUST be used only with v6 keys, as
+defined in [I-D.ietf-openpgp-crypto-refresh] Section 5.5.2.
```

Comment:
```suggestion
defined in [I-D.ietf-openpgp-crypto-refresh].
```
I think also 3.7.2.1 and 5.5.3 in crypto-refresh-08 applies here to the key material, maybe implicitely through the fingerprint also 5.1.2 and 5.2.3.35, and maybe some more. Shall we only cite the crypto-refresh and leave out the specific section. This seems to be more stable, since section numbering in the crypto-rtefresh can still change, and the key material packet information is somehow scattered throughout the crypto-refresh.